sessionctx/variable/sysvar.go

```diff
@@ -774,6 +774,10 @@ const (
 	SyncBinlog = "sync_binlog"
 	// BlockEncryptionMode is the name for 'block_encryption_mode' system variable.
 	BlockEncryptionMode = "block_encryption_mode"
+	// WaitTimeout is the name for 'wait_timeout' system variable.
+	WaitTimeout = "wait_timeout"
+	// WaitTimeoutDefaultValue is the default value of 'wait_timeout' system variable.
+	WaitTimeoutDefaultValue = "28800"
```

Comment:
Thanks for your quick fix, @hhxcc ! But we usually define the default values of system variable at 
https://github.com/pingcap/tidb/blob/e69aa2749b75d0608ccf062c771c63b737daa05c/sessionctx/variable/tidb_vars.go#L221

Reply:
ok ,  it has been moved to  tidb_vars.go